test/helpers/utils.ts

```diff
@@ -199,19 +201,25 @@ export async function setupOracle(
     value: parseEther("0.5"),
   });
 
-  await oracle
-    .connect(oracleOwnerSigner)
-    .setStablePrice(USDC_ADDRESS[chainId], "100000000");
+  // For example, we don't consider MIM to be stable, could de-peg
+  if (strikePricer) {
+    await oracle
+      .connect(oracleOwnerSigner)
+      .setStablePrice(MIM_ADDRESS[chainId], "100000000");
```

Comment:
TODO: Needs to be `setAssetPricer()` but some issues with setting the ChainLink Pricer for data in the future (that doesn't exist).